go/private/actions/library.bzl

```diff
@@ -26,7 +26,7 @@ load("@io_bazel_rules_go//go/private:providers.bzl",
     "searchpath_attr",
 )
 
-def emit_library(ctx, go_toolchain, srcs, deps, cgo_object, embed, want_coverage, importpath, golibs=[]):
+def emit_library(ctx, go_toolchain, srcs, pure_go_srcs, deps, cgo_object, embed, want_coverage, importpath, golibs=[]):
```

Comment:
I think I would probably swap these.
srcs should always be the original sources, the fact that we need to pass in cgo transformed ones is a temporary hack, and should be the extra parameter.